### ISSUE TITLE
[SNES Core feature request] Sound interpolation option

### ISSUE THREAD

**Issue by Yave-Yu** [TASEmulators/BizHawk#3221]
Is it possible to add sound interpolation option on all existing SNES cores?

None
Linear
Gaussian (Default)
Cubic
Sinc

Also BSNESv115+ didn't have graphic debugger that I can change palette to Snes9x to make color into 8x (248 white), so can you add this feature into it?

**Comment by nattthebear** [comment]
No, those sound interpolation options are stupid and worthless.  If you want the real original for some sort of precision work, you want https://github.com/TASEmulators/BizHawk/issues/3207.  Anything else would be a waste of our time.

**Comment by Yave-Yu** [comment]
> 

Just feel Gaussian eats high-freq is a bit awful, even though it's close to real hardware.

**Comment by Morilli** [comment]
BSNESv115+ does have an internal option for switching between gaussian and cubic interpolation methods, but from the quick manual testing I did I was not able to hear any sort of difference, so I never bothered exposing it as an option to the frontend.
